scripts/publicNames.txt

```diff
@@ -507,7 +500,9 @@ girder
         mongodb_proxy
             EXECUTABLE_MONGO_METHODS
             Executable
+            MongoClient
             MongoProxy
+            MongoReplicaSetClient
```

Comment:
I'm dubious that these belong here since they are spiritually the  same as imports, here's the code snippet in question:

```python
try:
    # will fail to import from older versions of pymongo
    from pymongo import MongoClient, MongoReplicaSetClient
except ImportError:
    MongoClient, MongoReplicaSetClient = None, None
```

Reply:
Yes, I've seen, and I'd consider it a false positive. However, the tool is emitting them as ordinary variables (without any additional type flags), so unless we want to build a whitelisting mechanism (at which point, we've started building our own linter from scratch), we are stuck with them.

The old tool was emitting keyword arguments as variables too, and nobody noticed. It seems like the point of the list is mostly to flag deltas, and is less so an absolute reference.

Reply:
> It seems like the point of the list is mostly to flag deltas, and is less so an absolute reference.

This is true. There will be a human in the loop to review if these change anyway.